presto-docs/src/main/sphinx/admin/web-interface.rst

```diff
@@ -16,7 +27,30 @@ running queries.
 When the server is configured to use HTTPS, a :doc:`password authenticator
 </develop/password-authenticator>` such as :doc:`LDAP </security/ldap>` or
 :doc:`password file </security/password-file>` must be configured in order to
-use the Web UI.
+use the Web UI. In these cases, the authentication type defaults to the required
+``form`` value. It triggers the Web UI to display a login form, that accepts the
+relevant username and password, and verifies against the authentication backend.
+
+If you require the Web UI to be accessible to anyone, you can set the
+authentication type to ``fixed`` and configure the ``web-ui.user``. As a result
+any interaction in the Web UI, such as killing a running query, is logged with
+the configured user.
+
+Alternatively the authentication type can be set to u se certificate
+authentication (``certificate``), :doc:`Kerberos authentication
+</security/server>` (``kerberos``), or JWT authentication (``jwt``).
+
+After successful login you have access to all information of queries submitted
+to Presto under the same username. You can view and kill your own queries.
```

Comment:
maybe combine these two sentence...  I'm thinking mixing the information in to the first part and not using a conjunction.

Reply:
done and simplified rest of paragraph as well